ci/get_features.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
```

Comment:
Now, this gets put back as `python`.

Reply:
I will drop this once if fixup the last commit, I was just gonna test if it works on Python2.

Reply:
I guess it might be interesting to check if it works on python 2, but there isn't really much reason to do so.